### ISSUE TITLE
Update kernel module blacklist

### ISSUE THREAD

**Issue by garrettr** [freedomofpress/securedrop#1109]
During an installation last week, we encountered an issue with the kernel module blacklist. The install was using the new generation of Intel NUCs ([NUC5i5RYK](http://www.amazon.com/dp/B00SD9ISIQ) and [NUC5i5RYH](http://www.amazon.com/dp/B00SD9IS1S/)). Unlike the previous generation of NUCs, which did not include wireless networking hardware by default, the new generation includes wireless networking hardware for Wifi and Bluetooth on the motherboard.

This means that Ubuntu running on the servers not only loaded the high-level kernel modules for wifi and bluetooth support (`iwlwifi` and `bluetooth`), it also loaded modules necessary for support on the specific (included) hardware: `iwlmvm` and `btusb`. When the `remove kernel modules` Ansible role ran, it failed with an error because it could not remove the top-level modules without removing their dependencies first.

A quickfix to get this working on the new hardware was to change `disabled_kernel_modules` in `group_vars/securedrop.yml` from:

```yml
disabled_kernel_modules:
  - bluetooth
  - iwlwifi
```

to:

```yml
disabled_kernel_modules:
  - btusb
  - bluetooth
  - iwlmvm
  - iwlwifi
```

The order of the modules is important! We need to make sure the the dependencies are removed prior to the target modules that depend on them.

This list is also likely specific to the new generation of Intel NUCs. If we want to support a wider variety of hardware, we may want to try being smart about removing kernel modules and their dependencies, e.g. something akin to this technique from [Stack Exchange](https://askubuntu.com/questions/317230/how-can-i-temporarily-disable-a-kernel-module).

Finally, we need to make sure this updated module blacklist still works on the old hardware as well.

**Comment by garrettr** [comment]
Confirmed that this proposed change to `disabled_kernel_modules` still leads to a successful install on the "old hardware" (Intel NUC D54250WYK used for the Application and Monitor servers).

**Comment by conorsch** [comment]
Given that we decided in #1112 to continue recommending the EOL'd NUCs for now, do we still want to target 0.3.5 with this fix?

**Comment by garrettr** [comment]
Merged into release/0.3.5 in #1116 .
